modules/core/server/views/layout.server.view.html

```diff
@@ -65,6 +65,11 @@
 
   <!--Application JavaScript Files-->
     {{#each jsFiles}}<script type="text/javascript" src="{{this}}"></script>{{/each}}
+  
+  <!--owasp config sync-->
+  <script type="text/javascript">
+    owaspPasswordStrengthTest.configs = {{{ owaspConfig }}};
```

Comment:
Good catch. It seems like setting configs directly works, but its better to set it via the function as per the docs.